packages/autocomplete-core/src/__tests__/getAutocompleteSetters.test.ts

```diff
@@ -51,19 +51,60 @@ describe('createAutocomplete', () => {
     });
   });
 
-  test('setSuggestions', () => {
-    const onStateChange = jest.fn();
-    const { setSuggestions } = createAutocomplete({
-      getSources: () => [],
-      onStateChange,
+  describe('setSuggestions', () => {
+    test('default', () => {
+      const onStateChange = jest.fn();
+      const { setSuggestions } = createAutocomplete({
+        getSources: () => [],
+        onStateChange,
+      });
+      const suggestions = [createSuggestion([{ item: 'hi' }])];
+
+      setSuggestions(suggestions);
```

Comment:
Any reasons why we use `setSuggestions` in this test and `refresh` in the other? If not, can we use `refresh` in the two for consistency?

Reply:
the second test is testing the internal use of setSuggestions more or less by using refresh. Should it be in a different describe maybe instead?

Reply:
Then, since we're in the "setSuggestions" `describe`, can we return an empty array in `getSources` and use `setSuggestions` in the test below as well?